pkg/controller/storagecluster/storagecluster.go

```diff
@@ -67,6 +67,7 @@ const (
 	defaultStorageClusterUniqueLabelKey = apps.ControllerRevisionHashLabelKey
 	defaultRevisionHistoryLimit         = 10
 	defaultMaxUnavailablePods           = 1
+	deleteFinalizerName                 = "libopenstorage.org/delete"
```

Comment:
Can you update the prefixes to `operator.libopenstorage.org`? It should have the project name too.